roles/tripleo_cluster_spec_render/defaults/main.yml

```diff
@@ -1,16 +1,14 @@
 ---
 # defaults file for tripleo_cluster_deploy
 tripleo_cephadm_services:
-  - {'service': 'mon', 'regex': '*controller*', 'label': 'mon'}
-  - {'service': 'mgr', 'regex': '*controller*', 'label': 'mgr'}
-  - {'service': 'osd', 'regex': '*ceph*', 'label': 'osd'}
+  - {'service_type': 'mon', 'service_id': 'mon', 'regex': '*controller*', 'label': 'mon'}
+  - {'service_type': 'mgr', 'service_id': 'mgr', 'regex': '*controller*', 'label': 'mgr'}
+  - {'service_type': 'osd', 'service_id': 'osd', 'regex': '*ceph*', 'label': 'osd'}
+  - {'service_type': 'nfs', 'service_id': 'nfs', 'regex': '*controller*', 'label': 'nfs',
```

Comment:
How often does service_type != service_id != label? 

If never, then why do we need three different variables to refer to the same thing?

Reply:
I'm basically fine with merging this provided that there's a good reason to have 3 variables, i.e. there are cases where they won't all be the same.

Reply:
for cephadm itself (not talking about tripleo-ceph), labels are user-defined without any defaults.

for `service_type != service_id`: 

* RGW uses `realm.zone` as `servcie_id`
* for cephfs, the service_id the name of the cephfs.